data/Mohiit70.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "Mohit Bisht",
+  "type": "personal",
+  "bio": "Passionate about Learning, Creating, and Contributing to the World of Web Development and Cloud.",
+  "followMe": [
```

It has to be links key.
```suggestion
  "links": [
```